src/coordinatorapi.cpp

```diff
@@ -546,6 +546,17 @@ DmtcpMessage CoordinatorAPI::sendRecvHandshake(DmtcpMessage msg,
       .Text("Connection rejected by the coordinator.\n"
             " Reason: This process has a different computation group.");
   }
+  // Coordinator also prints this, but its stderr may go to /dev/null
+  if (msg.type == DMT_REJECT_NOT_RESTARTING) {
+    string coordinatorHost = ""; // C++ magic code; "" to be invisibly replaced
+    int coordinatorPort;
+    Util::getCoordHostAndPort(COORD_ANY, coordinatorHost, &coordinatorPort);
+    JNOTE ("\n\n*** Computation not in RESTARTING or CHECKPOINTED state."
+        "\n***Reject incoming computation process requesting restart."
```

Comment:
Can you reword this comment? Maybe something like:

"Can't join the existing coordinator as it is serving a different computation. Consider launching a new coordinator."